### ISSUE TITLE
Error: Get http://localhost:8080/version: dial tcp [::1]:8080: connect: connection refused when installing connector

### ISSUE THREAD

**Issue by pierreroman** [virtual-kubelet/virtual-kubelet#184]
---

### Environment summary

Provider : ACI
K8s Master Info : AKS
Install Method : Helm Chart

### Issue Details
trying to deploy virtual-kubelet to AKS cluster in Azure from Windows 10 bash using azure-cli.

az aks install-connector \
    -n openhackk8s \
    -g openhack \
    --connector-name aci-connector

keep getting errors:

Merged "openhackk8s" as current context in /tmp/tmpx5h_q43x
Deploying the ACI connector for 'Linux' using Helm
**Error: Get http://localhost:8080/version: dial tcp [::1]:8080: connect: connection refused**

however, the cluster is running normally:

 kubectl config current-context
openhackk8s
kubectl get nodes
NAME                                      STATUS   ROLES   AGE     VERSION
aks-nodepool1-27269995-0   Ready     agent     7h        v1.8.11

root@SurfaceBook3:~# kubectl get pods
NAME                                      READY     STATUS    RESTARTS   AGE
minecraft-56ccd54b9-fv46d   1/1           Running   0                 4h
omsagent-54f6d                     1/1           Running   0                 6h

**Comment by robbiezhang** [comment]
please run `helm init` before running the install-connector command

**Comment by pierreroman** [comment]
i did that,  sorry i did not include it in the problem description

**Comment by robbiezhang** [comment]
Looks like a helm issue: https://github.com/kubernetes/helm/issues/3460

Can we take a look at the tiller log first?
```
kubectl get po -n kube-system | grep tiller | cut -d ' ' -f1 | xargs -I {} kubectl logs -n kube-system {}
```

**Comment by bacongobbler** [comment]
workaround is `helm init --service-account default` until we release v2.9.1.

**Comment by pierreroman** [comment]
had to tear down tiller.  re-install and it worked.  

closing this issue

**Comment by bacongobbler** [comment]
This has been fixed with helm v2.9.1, which was released yesterday. Please upgrade to get the fix or use the workarounds provided in kubernetes/helm#3985 if you're on v2.9.0. Thanks! :)
